### DISCUSSION TITLE
[announce] Check out a Tig extension plugin Tig Suite

### DISCUSSION THREAD

**Discussion by psprint** [universal-ctags/ctags#3630] [Show and tell]
A full featured Tig configuration that moves the shell code from any `sh -c` bindings into separate files, which allows much more sophisticated setup.

Current features:
- ctags browsing from `tig` (or `fzf`, etc.), selecting opens `$EDITOR`,
- execution in `tig` of any past `:!…` or `:exec …` commands from `tig`, choosing from `tig`  or any other utility like `fzf`,
- stash support with custom non-conventional features like applying the stash with `patch` utility,
- so called °features° that are groups of files, selectable with `tig` (`tig-pick` contrib script to be exact) or `fzf`, etc.

### Examples

Past commands history:

![2023-01-10-134324_1882x1044_scrot](https://user-images.githubusercontent.com/6049288/211555445-cb46713e-80d7-4985-b186-0a675bd15b3c.png)

Tags browser:

![2023-01-10-135829_1879x1040_scrot](https://user-images.githubusercontent.com/6049288/211558195-3c568103-5e4b-467c-b63c-cc93e046a024.png)

°Feature° list/open:

![2023-01-10-134421_1879x1044_scrot](https://user-images.githubusercontent.com/6049288/211555502-b4f4ad23-a47e-46b9-93ba-dc3ca3c94630.png)

Homepage: https://github.com/psprint/tigsuite

**Comment by masatake** [comment]
Can I move this to "discussion"?

**Comment by psprint** [comment]
Yes, go ahead
